style.css

```diff
@@ -0,0 +1,43 @@
+body {
+  background-color: mistyrose;
+  margin-left: 150px;
```

Comment:
- The display is one-sided because the margins don't correspond. Always style in a manner that contents and items will be either justified center, left right or float etc...